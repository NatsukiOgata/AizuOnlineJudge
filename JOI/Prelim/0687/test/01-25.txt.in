40
515 226 367 753 175 399 683 1129 1873 1544 56 256 1580 1627 1438 1789 286 1410 1479 983 1799 607 1917 1570 832 1237 222 907 252 1414 2000 1426 1946 1641 1238 216 1252 1152 72 1046
