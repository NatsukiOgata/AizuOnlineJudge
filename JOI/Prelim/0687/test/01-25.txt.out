29325
9989
